src/main/java/org/candlepin/service/impl/DefaultEntitlementCertServiceAdapter.java

```diff
@@ -399,29 +344,10 @@ private String getContentPrefix(Consumer consumer, Owner owner, boolean useConte
         return result;
     }
 
-    public Set<X509ByteExtensionWrapper> prepareV3ByteExtensions(Product sku,
-        List<org.candlepin.model.dto.Product> productModels, String contentPrefix,
-        Map<String, EnvironmentContent> promotedContent) throws IOException {
+    public Set<X509ByteExtensionWrapper> prepareV3ByteExtensions(
+        List<org.candlepin.model.dto.Product> productModels) throws IOException {
 
-        Set<X509ByteExtensionWrapper> result = v3extensionUtil.getByteExtensions(sku,
-            productModels, contentPrefix, promotedContent);
-        return result;
-    }
-
-    // Encode the entire prefix in case any part of it is not
-    // URL friendly. Any $ is put back in order to preseve
-    // the ability to pass $env to the client
-    public String cleanUpPrefix(String contentPrefix) throws IOException {
-
-
-        StringBuffer output = new StringBuffer("/");
-        for (String part : contentPrefix.split("/")) {
-            if (!part.equals("")) {
-                output.append(URLEncoder.encode(part, "UTF-8"));
-                output.append("/");
-            }
-        }
-        return output.toString().replace("%24", "$");
+        return v3extensionUtil.getByteExtensions(productModels);
```

Comment:
At this point, this method does nothing other than passing the single argument and returning, so we could remove it and just call the underlying method instead?

Reply:
Maybe? As pointless as it is, it's bridging the adapter interface for us. We may need to keep the implementation as it is, but we could probably change the code calling it to, instead, call the extension util method directly.

Otherwise, if we really want to remove it, I'd say we should look at all the cert service adapters and check with teamnado to make sure they don't have anything custom in this area. Katello probably as well, but I think we do all of the adapter management for them.